### PR TITLE
Some logging improvements

### PR DESCRIPTION
**Pull request by bostrt**
SPLAT-618

### CODE REVIEW

**Comment by bostrt** [comment]
The reason for [23891f3](https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/4/commits/23891f378b92540ad0d1183de31a872a997912db) is that I had introduced a `log.Fatal` (which exits process) and it was leading to commands like `version` and `assets` to fail even though they don't use kubeconfig.

**Comment by bostrt** [comment]
/cc @mtulio 

mind taking a look?

File: cmd/root.go [hunk 1]
```diff
@@ -5,41 +5,39 @@ import (
...
 		log.SetLevel(logrusLevel)
 
-		// Save kubeconfig
-		client.Kubeconfig = viper.GetString("kubeconfig")
-
-		return nil
+		// Additional log options
+		customFormatter := new(log.TextFormatter)
+		customFormatter.FullTimestamp = true
+		log.SetFormatter(customFormatter)
```
Comment (mtulio) [inline cmd/root.go:40]: Unless you need to use the `customFormatter`, you could use the inline instantiation like:
```suggestion
		log.SetFormatter(&log.TextFormatter{
		    FullTimestamp: true,
		})
```

**Review by mtulio — APPROVED** [review]
/lgtm

**Pull request merged**